packages/orbit-components/.storybook/orbitDecorator.tsx

```diff
@@ -1,13 +1,16 @@
 import * as React from "react";
 import jsxToString from "react-element-to-jsx-string";
-import { Source } from "@storybook/blocks";
+import { Highlight, themes } from "prism-react-renderer";
 
 import defaultTheme from "../src/defaultTheme";
 import OrbitProvider from "../src/OrbitProvider";
 import Heading from "../src/Heading";
 import Text from "../src/Text";
+import Tile from "../src/Tile";
 
 const OrbitDecorator = (storyFn, context) => {
+  const [isCopied, setIsCopied] = React.useState(false);
```

Comment:
I added this simple solution for the copying functionality. Prism renderer [doesn't offer any wrapper](https://github.com/FormidableLabs/prism-react-renderer/issues/181) and for our purposes this seems enough compared to adding extra package for handling copying. And Prism renderer seems to be the best package for the syntax highlight so this seemed reasonable to me.